### ISSUE TITLE
RHEL 7 - PAM rules gets fixed by remediation but fail again after reboot

### ISSUE THREAD

**Issue by mildas** [ComplianceAsCode/content#8302]
#### Description of problem:
PAM rules fail after kickstart installation and also when tested by SSGTS in profile mode.
During remediation phase, PAM rules are `fixed` and with evaluation message: `Fix execution completed and returned: 0`. However, after reboot they result in fail.

#### SCAP Security Guide Version:
4bca873

#### Operating System Version:
RHEL 7.9

#### Steps to Reproduce:

1. `python3 tests/test_suite.py profile --libvirt qemu:///system test_suite_vm --datastream /tmp/ssg-rhel7-ds.xml xccdf_org.ssgproject.content_profile_ospp`

or

1. Install using kickstart (e..g ANSSI High)
2. Scan after installation

or

1. On clean RHEL 7.9, `oscap xccdf eval --profile pci-dss --remediate ssg-rhel7-ds.xml`
2. e.g. `oscap xccdf eval --profile '(all)' --rule xccdf_org.ssgproject.content_rule_display_login_attempts ssg-rhel7-ds.xml`

#### Actual Results:
PAM rules fail

#### Expected Results:
PAM rules pass

#### Additional Information/Debugging Steps:
In the final HTML report, there is no `pass` result in OVAL details.

If you run `oscap xccdf eval --remediate` second time, e.g. `oscap xccdf eval --profile pci-dss --remediate ssg-rhel7-ds.xml`, then it's remediation phase looks like this (smartcard_auth error is not relevant):
```
Title   Ensure PAM Displays Last Logon/Access Notification
Rule    xccdf_org.ssgproject.content_rule_display_login_attempts
Ident   CCE-27275-7
Result  fixed

Title   Limit Password Reuse
Rule    xccdf_org.ssgproject.content_rule_accounts_password_pam_unix_remember
Ident   CCE-82030-8
Result  fixed

Title   Enable Smart Card Login
Rule    xccdf_org.ssgproject.content_rule_smartcard_auth
Ident   CCE-80207-4
Result  error
```
2 PAM rules got fixed. Now, after the second remediation, check them individually:
```
# oscap xccdf eval --profile '(all)' --rule xccdf_org.ssgproject.content_rule_display_login_attempts ssg-rhel7-ds.xml
WARNING: Datastream component 'scap_org.open-scap_cref_security-data-oval-com.redhat.rhsa-RHEL7.xml.bz2' points out to the remote 'https://access.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2'. Use '--fetch-remote-resources' option to download it.
WARNING: Skipping 'https://access.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2' file which is referenced from datastream
WARNING: Skipping ./security-data-oval-com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Title   Ensure PAM Displays Last Logon/Access Notification
Rule    xccdf_org.ssgproject.content_rule_display_login_attempts
Ident   CCE-27275-7
Result  pass

# oscap xccdf eval --profile '(all)' --rule xccdf_org.ssgproject.content_rule_accounts_password_pam_unix_remember ssg-rhel7-ds.xml
WARNING: Datastream component 'scap_org.open-scap_cref_security-data-oval-com.redhat.rhsa-RHEL7.xml.bz2' points out to the remote 'https://access.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2'. Use '--fetch-remote-resources' option to download it.
WARNING: Skipping 'https://access.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2' file which is referenced from datastream
WARNING: Skipping ./security-data-oval-com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Title   Limit Password Reuse
Rule    xccdf_org.ssgproject.content_rule_accounts_password_pam_unix_remember
Ident   CCE-82030-8
Result  fail
```
The PAM rules are breaking each other.

**Comment by mildas** [comment]
Fails also with ansible remediation using `--remediate-using ansible` option.

**Comment by yuumasato** [comment]
Should be fixed by #8347
